_includes/_header.htm

```diff
@@ -1,18 +1,20 @@
-<nav class="navbar navbar-toggleable-sm fixed-top navbar-transparent" color-on-scroll="75">
-    <div class="container" role="navigation">
+<nav class="navbar navbar-expand-lg fixed-top">
+    <div class="container justify-content-center">
         <div class="row">
-            <div>
-                <a class="navbar-brand" href="/index.htm" rel="tooltip" title="Gridcoin - Securely rewarding BOINC computation on top of POS!" data-placement="bottom"><img src="/assets/img/grc-title2.png" alt="Gridcoin title" /></a>
-            </div>
-
-            <div class="btn-group" >
-
-                <div class="dropdown button-dropdown invert">
-                    <a href="#pablo" class="dropdown-toggle" id="navbarDropdown" data-toggle="dropdown">
-                        <button class="btn btn-round">Community <i class="now-ui-icons arrows-1_minimal-down"></i></button>
-                    </a>
-                    <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-                        <a class="dropdown-header">Gridcoin Community</a>
+            <a name="Nav"></a> <!-- Nav link, returning user to top of website -->
+            <a class="navbar-brand" href="/index.htm" rel="tooltip" title="Gridcoin - Securely rewarding BOINC computation on top of POS!" data-placement="bottom"><img src="/assets/img/grc-title2.png" alt="Gridcoin title" /></a>
+            <button class="navbar-toggler grcbtn" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="oi oi-menu"></span>

```

Comment:
Minor suggestion: we can use Bootstrap's built-in `navbar-toggler-icon` on this `<span>` instead. The SVG renders more crisply than `oi-menu`. Add the `navbar-dark` class to the `navbar-toggler` button for the image to appear.